chapters/ch04.md

```diff
@@ -1,6 +1,6 @@
 > # 4. Scale drives machine learning progress
 
-# 4. Quy mô quyết định mô hình machine learning
+# 4. Quy mô định hướng quá trình phát triển machine learning
```

Comment:
Ý của tác giả trong chương này theo mình hiểu là quá trình phát triển của cả lĩnh vực machine learning nói chung được tạo ra bởi hai nhân tố chính. Machine learning progress không ám chỉ riêng một mô hình machine learning nào cả.

@ngcthuong @william-vu @sonvx vào thảo luận thêm.

Reply:
Tiêu đề nay ở PR cũ em đã muốn dịch theo hướng là quy mô (về data, năng lực tính toán) có sự kích thích, thúc đẩy một cách đột biến sự phát triển của ML.   

> Many of the ideas of deep learning (neural networks) have been around for decades. Why are these ideas taking off now?

Vì thế e nghĩ từ "định hướng" có vẻ hơi "nhẹ", có lẽ là: 

> Quy mô thúc đẩy sự phát triển của ML

Reply:
@SumoBBQ anh thích cách dịch của em. Chờ thêm một vài ý kiến nữa rồi mình merge.

Reply:
Mình đã sửa thành:
> Quy mô thúc đẩy quá trình phát triển machine learning